PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/components/ServerExplorerToolWindowFactory.java

```diff
@@ -205,6 +204,15 @@ private void treeMousePressed(MouseEvent e, JTree tree) {
         }
     }
 
+    private Node getTreeNodeOnMouseClick(JTree tree, TreePath treePath) {
```

Comment:
static?

Reply:
the class is used with its instance, no need to make it as static.